modules/alloy/src/main/ts/ephox/alloy/behaviour/disabling/ActiveDisable.ts

```diff
@@ -12,7 +12,7 @@ import { DisableConfig } from './DisableTypes';
 const exhibit = (base: DomDefinitionDetail, disableConfig: DisableConfig) => DomModification.nu({
   // Do not add the attribute yet, because it will depend on the node name
   // if we use "aria-disabled" or just "disabled"
-  classes: disableConfig.disabled ? disableConfig.disableClass.map(Arr.pure).getOr([ ]) : [ ]
+  classes: disableConfig.disabled() ? disableConfig.disableClass.map(Arr.pure).getOr([ ]) : [ ]
```

Comment:
Is `disableConfig.disableClass.map(Arr.pure).getOr([ ])` just `disableConfig.disableClass.toArray()`?

Reply:
Yeah, that should be the same, as `disableClass` is `Option<string>`. I really only wanted to touch a minimal amount in this PR though, as it was just meant to be a quick backport to fix bugs, so I really didn't look at other issues. Having said that though, I'll go ahead and fix this.

Reply:
Fixed